README.md

```diff
@@ -116,6 +116,18 @@ Install all the APKs, then start the Kitkit Launcher to explorer.
 - Please make sure there is enough storage on the device. 
 - Most of the APKs will not show up in the android (default) launcher. 
 
+
+# Trademarks
+Kitkit School, name and logo, and Enuma are trademarks of Enuma, Inc. The Apache License 2.0 under which Kitkit School software is released does not grant any license or right to use the trademarks. Please see [Guidelines for Trademarks](http://kitkitschool.com/legal-kitkit/) to learn more about the terms. 
```

Comment:
@todoschool Can you include the logo here so that people will know what it looks like?